device/utilities/network/generic_network_utility.py

```diff
@@ -0,0 +1,169 @@
+import os, subprocess
+from typing import List, Dict
+
+from device.utilities.network.base_network_utility import NetworkUtility
+
+
+class GenericNetworkUtility(NetworkUtility):
+    def __init__(self) -> None:
+        super(GenericNetworkUtility, self).__init__()
+        self.GET_WIFI_SSIDS_SCRIPT_PATH = "scripts/network/get_wifi_ssids.sh"
```

Comment:
Is this for backwards compatibility and/or OSX support?    If so, then its fine.   But I was thinking we get rid of those scripts for all platforms and use specialized classes that do what is in the scripts using subprocess.run()

Reply:
Exactly. I needed a place to put what we had until we replaced it